site/soc/software/charts/osh/openstack-mariadb/mariadb.yaml

```diff
@@ -32,8 +32,6 @@ data:
     pod:
       resources:
         enabled: {{ openstack_helm_pod_resources_enabled['mariadb'] }}
-      replicas:
```

Comment:
only removing this is not enough because on line 22 we are replacing the `.values.pod.replicas.server` from the chart with our values from `.pods.osh.mariadb.server.min` which comes from the pod scale profile (in our case, the default pod-scale-mininal.yaml)

In order to increase this to 3, we need to change that value in the pod scale profile

Reply:
Thanks, good point!

However I think we do not want to change the definition of `pod-scale-minimal.yaml` profile, to keep it minimal, right? So here what should be enough is to remove this value replace ... I hope

Reply:
yep, that should work but....then you are removing the user being able to pass a custom profile for the number of pods, so they cant configure if they want just 1 pod or 30, which is bad IMO, especially becuase we still have that `.pods.osh.mariadb.server` option in both scale profiles (minimal and ha)

Reply:
Ok, then I should point to different profile. That has to be the right solution.

Reply:
Update: it is not. I cannot point to another profile while another one is being used as a default